CONTRIBUTING.md

```diff
@@ -97,24 +97,38 @@ Mitiq code is developed according the best practices of Python development.
 
+#### Some possible issues with `Black`
+Below is a summarized list of some issues leading to errors. A more detailed discussion can be found [in the documentation](https://black.readthedocs.io/en/stable/the_black_code_style.html#the-black-code-style).
+- In general, `Black` strives for one expression or statement per line with uniform vertical and horizontal whitespace. This is described as `Black` doing everything to keep "[`pycodestyle` happy](https://black.readthedocs.io/en/stable/the_black_code_style.html#how-black-wraps-lines)". A list of errors due to `pycodestyle` can be found [here](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes).
+- When a statement or expression is moved to its own line, a [trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style.html#trailing-commas) will be added.
+- Because uniform vertical whitespace is enforced, empty lines are only allowed in a [handful of scenarios](https://black.readthedocs.io/en/stable/the_black_code_style.html#empty-lines) like inside, before and after function definitions etc.
+- [Default line length](https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length) for `Black` is 88 characters per line. This number can be adjusted to have fewer or more characters as needed. As stated above, default length for `Mitiq` is 79 characters.
+- If an expression or a statement can be fit in one line, trailing comma introduced before a line break will be removed along with the [parentheses](https://black.readthedocs.io/en/stable/the_black_code_style.html#parentheses).
+- If backslashes are used for line breaks, these will be replaced with parentheses in keeping with [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).
+- To obey [PEP 8 style guide for improving readability](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator), a line break will be introduced before a binary operator.
+- Use double quotes for [strings](https://black.readthedocs.io/en/stable/the_black_code_style.html#strings).
+
+
 ## Proposing a new feature to Mitiq
 
 If you are interested in adding a larger new feature or functionality to Mitiq, please check out our
 Mitq enhancement proposal (MEP) template [`docs/mep/0000-feature-proposal-TEMPLATE.md`](https://github.com/unitaryfund/mitiq/blob/master/docs/mep/0000-feature-proposal-TEMPLATE.md). To help facilitate
 discussion about the feature you would like to add, make a copy of the template and increment the proposal
 number and change `feature-proposal-TEMPLATE` to a short description of what you are proposing.
-Please fill out any relevant sections of that template as best you can and we can discuss in 
+Please fill out any relevant sections of that template as best you can and we can discuss in
 both PR threads as well as on the [discord](https://discord.unitary.fund).
```

Comment:
Just a note if you want to fix it here: https://discord.unitary.fund is not a clickable link. One has to copy/paste discord.unitary.fund into the browser.